python/paddle/fluid/__init__.py

```diff
@@ -99,8 +99,8 @@ def __bootstrap__():
         None
     """
     import sys
-    import core
     import os
+    from . import core
```

Comment:
why?

Reply:
relative import is not support in python3